From bb414b02a9474f0acf5231afb5a954bda0bab2fe Mon Sep 17 00:00:00 2001
From: SuperSix173 <liuchao173@huawei.com>
Date: Sat, 31 Jul 2021 15:29:09 +0800
Subject: [PATCH] skip check_web testcase

check_web needs to access validator.w3.org. Therefore, make check
fails because the external network cannot be accessed.

---
 Makefile     | 2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1136af9..5ccad33 100644
--- a/Makefile
+++ b/Makefile
@@ -756,7 +756,7 @@ tzselect:	tzselect.ksh version
 
 check:		check_character_set check_white_space check_links \
 		  check_name_lengths check_sorted \
-		  check_tables check_web check_zishrink check_tzs
+		  check_tables check_zishrink check_tzs
 
 check_character_set: $(ENCHILADA)
 	test ! '$(UTF8_LOCALE)' || \
-- 
1.8.3.1

